src/main/java/org/folio/circulation/resources/RequestByInstanceIdResource.java

```diff
@@ -209,11 +202,12 @@ private void createInstanceLevelRequests(RoutingContext routingContext) {
           && (instanceRequestPackage.getSortedAvailableItems() == null || instanceRequestPackage.getSortedAvailableItems().isEmpty())) {
           //fail the requests when there are no items to make requests from.
           log.error("Failed to find request queues for all items of instanceId {}",
-            unsortedUnavailableItems.get(0).getInstanceId());
+            items.toArray(new Item[0])[0].getInstanceId());
```

Comment:
I know this is just an error message, but this code makes me sad. 😄 How about:
```suggestion
            items.iterator().next().getInstanceId());
```
Or better yet, can't we just get the instance id from the `instanceRequestPackage`:
```suggestion
           instanceRequestPackage.getInstanceLevelRequest().getInstanceId().toString());
```

Reply:
indeed. Used  instanceRequestPackage.getInstanceLevelRequest().getInstanceId()